subprojects/core/src/test/groovy/org/gradle/api/internal/tasks/DefaultTaskInputsTest.groovy

```diff
@@ -198,7 +198,7 @@ class DefaultTaskInputsTest extends Specification {
         when: inputs.files(["s1", "s2"]).skipWhenEmpty()
         then:
         inputs.hasSourceFiles
-        inputs.files.files.toList() == [new File("a"), new File("b"), new File("s1"), new File("s2")]
+        inputFiles().toList().sort() == [new File("a"), new File("b"), new File("s1"), new File("s2")]
```

Comment:
Would it make sense for `inputFiles()` to return a sorted list by default?

Reply:
done.